lib/commands/smapi/smapi-command-handler.js

```diff
@@ -60,6 +60,8 @@ const _mapToParams = (optionsValues, flatParamsMap, commanderToApiCustomizationM
     return res;
 };
 
+const _sdkFunctionName = (swaggerApiOperationName) => `call${swaggerApiOperationName.charAt(0).toUpperCase() + swaggerApiOperationName.slice(1)}`;
```

Comment:
if we call function that starts with call* it has additional metadata in the response.